terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
@bttanakaysd 
You mean to fix `5.3.x` branch in the same way?
I think that the maintenance branches must not be changed and that the master branch should be fixed based on what it should be. Of course, whether my proposal is good is another matter.

And I had understood as follows.
* `logServletException` mainly handle the fatal errors not handled by `HandlerExceptionResolver`.
* `logIOException` mainly handle the errors occurred with writing response in view and filter.
* `logRuntimeException` handle other errors. (**Normally, it should not occur.**)

If they are designed to be separately extensible because they have different roles, it would be better to include that in JavaDoc.

Thanks.